packages/renderer/src/lib/task-manager/TaskManagerEmptyFilteredScreen.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import { FilteredEmptyScreen } from '@podman-desktop/ui-svelte';
+
+import TaskIcon from '/@/lib/images/TaskIcon.svelte';
+
+let { searchTerm = $bindable(), onReset = () => {} }: { searchTerm: string; onReset?: (term: string) => void } =
```

Comment:
suggestion (non-blocking): might create a Props above when we have more that one?

Reply:
nice usage of svelte5 btw

Reply:
fixed